_posts/en/newsletters/2021-05-19-newsletter.md

```diff
@@ -0,0 +1,157 @@
+---
+title: 'Bitcoin Optech Newsletter #149'
+permalink: /en/newsletters/2021/05/19/
+name: 2021-05-19-newsletter
+slug: 2021-05-19-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter provides updates on the previously proposed
+transaction relay reliability workshop and CVE-2021-31876.  Also
+included are our regular sections describing updates to services and
+client software, new releases and release candidates, and notable
+changes to popular Bitcoin infrastructure software.
+
+## News
+
+- **Relay reliability workshop scheduled:** as mentioned in [Newsletter
+  #146][news146 workshop], Antoine Riard will be hosting IRC-based
+  meetings to discuss how to make unconfirmed transaction relay more
+  reliable for contract protocols such as LN, coinswaps, and DLCs.  The
+  schedule is,
+
+    - Jun 15th, 19:00--20:30 UTC: guidelines about L2 protocols onchain
+      security design; Coordinated cross-layers security disclosures;
+      full-RBF proposal
+
+    - June 22nd (same time): Generic layer two fee-bumping primitive
+      (such as package relay)
+
+    - June 29th (same time): reserved for additional discussions.
+
+- **CVE-2021-31876 BIP125 implementation discrepancy follow up:**
+  after the publication of [last week's newsletter][news148 cve], there was
+  additional discussion about the discrepancy between [BIP125][] opt-in
+  Replace-by-Fee ([RBF][topic rbf]) and Bitcoin Core's implementation.
+  Olaoluwa Osuntokun [confirmed][btcd #1719] that the BTCd full node
+  implemented BIP125 as specified, meaning it does allow child
+  transactions to be replaced based on inherited signaling.  Ruben
+  Somsen noted that a hypothetical variation of spacechains, a type of
+  one-way pegged [sidechain][topic sidechains], would be affected by the
+  problem.  On the other hand, Antoine “Darosior” Poinsot noted that the
+  Revault [vault][topic vaults] architecture wouldn't be affected.
+
+## Changes to services and client software
+
+*In this monthly feature, we highlight interesting updates to Bitcoin
+wallets and services.*
+
+- **Blockchain.com supports segwit:**
+  [v4.49.1][blockchain v4.49.1] of Blockchain.com's wallet adds the ability to
+  create a wallet with native segwit send and receive support.
+
+- **Sparrow 1.4.0 released:**
+  [Sparrow 1.4.0][sparrow 1.4.0] adds the ability to create a [child-pays-for-parent
+  (CPFP)][topic cpfp] transaction from the transaction list screen, user-defined
+  fee amounts during coin selection, and various other improvements.
+
+- **Electrum 4.1.0 enhances Lightning features:**
+  [Electrum 4.1.0][electrum 4.1.0] adds [trampoline payments][topic trampoline payments], [multipath
+  payment][topic multipath payments] support, the ability to perform channel
+  backups, and other Lightning features. Additionally, this version of Electrum
```

Comment:
"support" and "the ability to perform" seem redundant.

```suggestion
  [Electrum 4.1.0][electrum 4.1.0] adds [trampoline payments][topic trampoline payments], [multipath
  payments][topic multipath payments], channel
  backups, and other Lightning features. Additionally, this version of Electrum
```

Reply:
Updated!